### ISSUE TITLE
Error access server

### ISSUE THREAD

**Issue by kbariotis** [npm/newww#937]
I downloaded the `master` branch and started `couchdb`, `ellasticsearch` and `redis`. After running `npm run dev` I get:
```
npm run dev           (master⚡)

> newww@2.1.0 dev /Users/Bariotis/www/newww
> gulp dev

[20:18:00] Using gulpfile ~/www/newww/gulpfile.js
[20:18:00] Starting 'fonts'...
[20:18:00] Finished 'fonts' after 11 ms
[20:18:00] Starting 'images'...
[20:18:00] Finished 'images' after 2.51 ms
[20:18:00] Starting 'misc'...
[20:18:00] Finished 'misc' after 1.05 ms
[20:18:00] Starting 'styles'...
[20:18:00] Finished 'styles' after 3.48 ms
[20:18:00] Starting 'browserify'...
[20:18:00] Finished 'browserify' after 27 ms
[20:18:00] Starting 'concat'...
[20:18:00] Finished 'concat' after 9.95 ms
[20:18:00] Starting 'build'...
[20:18:00] Finished 'build' after 5.2 μs
[20:18:00] Starting 'nodemon'...
[20:18:00] Finished 'nodemon' after 1.58 ms
[20:18:00] Starting 'watch'...
[20:18:00] Finished 'watch' after 33 ms
[20:18:00] Starting 'dev'...
[20:18:00] Finished 'dev' after 5.47 μs
[gulp] [nodemon] v1.3.7
[gulp] [nodemon] to restart at any time, enter `rs`
[gulp] [nodemon] watching: *.*
[gulp] [nodemon] starting `node server.js`
2015-05-03T17:18:04.231Z  info server: server repl socket at /tmp/rpl/www-undefined.sock
2015-05-03T17:18:05.300Z  info server: Hapi server started @ https://localhost:15443
2015-05-03T17:18:14.921Z  info cache: get: https://127.0.0.1/package?sort=modified
2015-05-03T17:18:14.926Z  info cache: get: https://127.0.0.1/package?sort=dependents
2015-05-03T17:18:14.929Z  info cache: get: http://127.0.0.1:5984/package/-/count
2015-05-03T17:18:14.956Z error EXTERNAL: {
  "time": "2015-05-03T17:18:14.956Z",
  "hostname": "Kostass-MacBook-Pro.local",
  "pid": 31615,
  "level": "error",
  "name": "EXTERNAL"
}
2015-05-03T17:18:14.956Z error EXTERNAL: Error: connect ECONNREFUSED
  at exports._errnoException (util.js:746:11)
  at TCPConnectWrap.afterConnect [as oncomplete] (net.js:983:19)

2015-05-03T17:18:14.958Z error EXTERNAL: {
  "time": "2015-05-03T17:18:14.958Z",
  "hostname": "Kostass-MacBook-Pro.local",
  "pid": 31615,
  "level": "error",
  "name": "EXTERNAL"
}
2015-05-03T17:18:14.958Z error EXTERNAL: Error: connect ECONNREFUSED
  at exports._errnoException (util.js:746:11)
  at TCPConnectWrap.afterConnect [as oncomplete] (net.js:983:19)

Unhandled rejection Error: connect ECONNREFUSED
  at exports._errnoException (util.js:746:11)
  at TCPConnectWrap.afterConnect [as oncomplete] (net.js:983:19)
```

**Comment by zeke** [comment]
See https://github.com/npm/newww/issues/761

**Comment by alrra** [comment]
@ceejbot, @rockbot Maybe this should be closed in favor of #761?
